src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1562,7 +1562,7 @@ internal override ISymbolInternal CommonGetSpecialTypeMember(SpecialMember speci
         internal TypeSymbol GetTypeByReflectionType(Type type, BindingDiagnosticBag diagnostics)
         {
             var result = Assembly.GetTypeByReflectionType(type, includeReferences: true);
-            if ((object)result == null)
+            if (result is null)
```

Comment:
>if (result is null) [](http://example.com/codeflow?start=12&length=19)

Is this a "style-only" change? Please revert if so.

Reply:
A change of some sort is necessary to `(object?)` or use `is`. Personally, I'm happy with the latter.

Reply:
As Julien said, these lines need to change in some way to avoid the nullable warning for casting to `object`. Given that, I used the style of null check I prefer.